### PR TITLE
Use `?` to check for complex urls as it's more accurate than `$`

### PR DESCRIPTION
**Pull request by Page-**
Change-type: patch

### CODE REVIEW

File: src/sbvr-api/uri-parser.ts [hunk 1]
```diff
@@ -89,7 +89,7 @@ export const memoizedParseOdata = (() => {
 		max: env.cache.parseOData.max,
 	});
 	return (url: string) => {
-		if (_.includes(url, '$')) {
+		if (_.includes(url, '?')) {
```
Comment (dfunckt) [inline src/sbvr-api/uri-parser.ts]: Don't we also want to skip caching plain requests such as `/v5/device(12345)`?

Comment (Page-) [reply src/sbvr-api/uri-parser.ts]: Potentially but I think this covers the most important cases (based on checking cache entries in the api).  I also wonder whether with the much faster odata-parser if it's worth having this cache at all but that's a separate discussion

**Comment by Page-** [comment]
@resin-ci retest

**Comment by Page-** [comment]
@resin-ci retest

**Comment by Page-** [comment]
@resin-ci retest

**Comment by Page-** [comment]
@resin-ci retest

**Comment by nazrhom** [comment]
@resin-ci retest

**Comment by Page-** [comment]
@resin-ci retest

**Pull request merged**